### PR TITLE
cgame: use easier-to-translate strings with less argument-ordering problems

### PR DESCRIPTION
**Pull request by illwieckz**
cgame: use easier-to-translate strings with less argument-ordering problems

Obsolete:

- https://github.com/Unvanquished/Unvanquished/pull/2885

### CODE REVIEW

File: src/cgame/cg_event.cpp [hunk 1]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 			case MOD_LEVEL2_CLAW:
-				message = G_( "%s%s ^*was shredded by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL2;
+				message = G_( "%s%s ^*was shredded by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s claws; %s%s%s^* assisted" );
 				break;
 
 			case MOD_LEVEL2_ZAP:
-				message = G_( "%s%s ^*was electrocuted by %s%s%s^*'s %s" );
```
Comment (DolceTriade) [inline src/cgame/cg_event.cpp:488]: if you remove this, do you have to remove an argument elsewhere?

Comment (illwieckz) [reply src/cgame/cg_event.cpp:488]: Yes, the (now removed) `attackerClass` class variable and the (now removed) `if/else` code to duplicate the log messaging with it.

Comment (illwieckz) [reply src/cgame/cg_event.cpp:488]: This happens below in the diff.

**Review by DolceTriade — APPROVED** [review]

File: src/cgame/cg_event.cpp [hunk 2]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
 				break;
 
 			case MOD_LEVEL1_CLAW:
-				message = G_( "%s%s ^*was sliced by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL1;
+				message = G_( "%s%s ^*was sliced by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s claws; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:476]: Shouldn't we maintain consistency with current behaviour? In which case, this should say:

`"%s%s ^*was sliced by %s%s%s^*'s Mantis"`

File: src/cgame/cg_event.cpp [hunk 3]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL2_CLAW:
-				message = G_( "%s%s ^*was shredded by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL2;
+				message = G_( "%s%s ^*was shredded by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s claws; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:481]: Should be: `"%s%s ^*was shredded by %s%s%s^*'s Marauder"`

File: src/cgame/cg_event.cpp [hunk 4]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL2_ZAP:
-				message = G_( "%s%s ^*was electrocuted by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was electrocuted by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL2;
+				message = G_( "%s%s ^*was electrocuted by %s%s%s^*'s zap" );
+				messageAssisted = G_( "%s%s ^*was electrocuted by %s%s%s^*'s zap; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:486]: Should be: `"%s%s ^*was electrocuted by %s%s%s^*'s Marauder"`

File: src/cgame/cg_event.cpp [hunk 5]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL3_CLAW:
-				message = G_( "%s%s ^*was eviscerated by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was eviscerated by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL3;
+				message = G_( "%s%s ^*was eviscerated by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was eviscerated by %s%s%s^*'s claws; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:491]: Should be: `"%s%s ^*was eviscerated by %s%s%s^*'s Dragoon"`

File: src/cgame/cg_event.cpp [hunk 6]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL3_POUNCE:
-				message = G_( "%s%s ^*was pounced upon by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was pounced upon by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL3;
+				message = G_( "%s%s ^*was pounced upon by %s%s%s^*'s pounce" );
+				messageAssisted = G_( "%s%s ^*was pounced upon by %s%s%s^*'s pounce; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:496]: Should be: `"%s%s ^*was pounced upon by %s%s%s^*'s Dragoon"`

File: src/cgame/cg_event.cpp [hunk 7]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL3_BOUNCEBALL:
-				message = G_( "%s%s ^*was barbed by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was barbed by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
+				message = G_( "%s%s ^*was barbed by %s%s%s^*'s barb" );
+				messageAssisted = G_( "%s%s ^*was barbed by %s%s%s^*'s barb; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:501]: Should be: `"%s%s ^*was barbed by %s%s%s^*'s Dragoon"`

File: src/cgame/cg_event.cpp [hunk 8]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				messageSuicide = G_( "%s%s ^*was barbed" );
-				attackerClass = PCL_ALIEN_LEVEL3;
 				break;
 
 			case MOD_LEVEL4_CLAW:
-				message = G_( "%s%s ^*was mauled by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was mauled by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL4;
+				message = G_( "%s%s ^*was mauled by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was mauled by %s%s%s^*'s claws; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:507]: Should be: `"%s%s ^*was mauled by %s%s%s^*'s Tyrant"`

File: src/cgame/cg_event.cpp [hunk 9]
```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
...
 				break;
 
 			case MOD_LEVEL4_TRAMPLE:
-				message = G_( "%s%s ^*should have gotten out of the way of %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*should have gotten out of the way of %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL4;
+				message = G_( "%s%s ^*should have gotten out of the way of %s%s%s^*'s trample" );
+				messageAssisted = G_( "%s%s ^*should have gotten out of the way of %s%s%s^*'s trample; %s%s%s^* assisted" );
```
Comment (cu-kai) [inline src/cgame/cg_event.cpp:512]: Should be: `"%s%s ^*should have gotten out of the way of %s%s%s^*'s Tyrant"`

**Review by cu-kai — CHANGES_REQUESTED** [review]

**Comment by cu-kai** [comment]
ouch, merged while I was writing the review

**Comment by illwieckz** [comment]
I can do that, no problem. 😄️

**Comment by illwieckz** [comment]
@cu-kai:

- https://github.com/Unvanquished/Unvanquished/pull/2949

**Pull request merged**